### ISSUE TITLE
Releasenotes Heartcore

### ISSUE THREAD

**Issue by PascalIcatt** [umbraco/Umbraco.Cloud.Issues#567]
### Issue description

I noticed there is no release notes section for Heartcore like there is for [cloud](https://our.umbraco.com/documentation/Umbraco-Cloud/release-notes/)
Yesterday all our Heartcore sites have been upgraded, which happened flawlessly, and I would love to know about the new functionality that has been added/changed.

For instance, I noticed we now have a functioning deploy section with 3 options, this section used to be empty if I am correct.
Is there any documentation about the version (or the dll's used) deployed?

**Comment by sitereactor** [comment]
Yes, it makes a lot of sense to get release notes added so there is something to refer to for the updates we roll out.
We'll see if we can get the ball rolling on this next week.
